src/main/java/intro/bookservice/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package intro.bookservice.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
```

Comment:
ISBN has other validation rules too. You can use predefined annotation bundled in bean validation API.